pbs/pbsrequest.go

```diff
@@ -30,20 +37,42 @@ type Bids struct {
 	Params     json.RawMessage `json:"params"`
 }
 
+type PBSVideo struct {
```

Comment:
yes, please :). It's not the only one... I added some godocs in #77, and would have loved to add them to the whole project... but I could only squeeze in so much time on it before other projects came back up.

Any time spent writing a doc is multiplied by everyone who reads it... and code is always read more times than it's written. The strongest engineers I've heard on the subject recommend "at minimum, document every public property, function, method, argument, return value, and precondition."